content/influxdb/cloud/reference/cli/influx/_index.md

```diff
@@ -12,5 +12,106 @@ weight: 101
 influxdb/cloud/tags: [cli]
 ---
 
-{{< duplicate-oss >}}
+The `influx` command line interface (CLI) includes commands to manage many aspects of InfluxDB,
+including buckets, organizations, users, tasks, etc.
 
+For information about setting up and configuring the `influx` CLI, see [Download, install, and use the influx CLI](/influxdb/cloud/sign-up/#optional-download-install-and-use-the-influx-cli).
+
+## Usage
+
+```
+influx [flags]
+influx [command]
+```
+
+### Provide required authentication credentials
+To avoid having to pass your InfluxDB **host**, **authentication token**, and **organization**
+with each command, store them in an `influx` CLI configuration (config).
+`influx` commands that require these credentials automatically retrieve these
+credentials from the active config.
+
+Use the [`influx config create` command](/influxdb/cloud/reference/cli/influx/config/create/)
+to create an `influx` CLI config and set it as active:
+
+```sh
+influx config create --config-name <config-name> \
+  --host-url http://localhost:8086 \
+  --org <your-org> \
+  --token <your-auth-token \
+  --active
+```
+
+For more information about managing CLI configurations, see the
+[`influx config` documentation](/influxdb/cloud/reference/cli/influx/config/).
+
+## Commands
+
+| Command                                                      | Description                                          |
+|:-------                                                      |:-----------                                          |
+| [apply](/influxdb/v2.0/reference/cli/influx/apply)           | Apply an InfluxDB template                           |
```

Comment:
should these point to cloud URLs? There is functionality in the `influx` tool that is specific to OSS and Cloud.